daq/panda/testing/daqTestScript.py

```diff
@@ -56,58 +56,60 @@
   now = datetime.now() #set time
   timeName = now.strftime("%H:%M:%S:%f") [:-3] #set current time
 
+  numValues = 22; #expected number of data inputs
```

Comment:
May be nice to make this a constant that you declare at the very top of the script

Reply:
Apparently it's [not possible](https://stackoverflow.com/questions/2682745/how-do-i-create-a-constant-in-python)? Also, I thought it was better to have the number close to the loop, so that the code is more easily readable

Reply:
Not a declared constant but just something like NUM_VALUES = 22. Better to leave important global variables like that at the top. 

Also, why are you putting semicolons in the python code? Im pretty sure that will throw an error?